### PR TITLE
Add snneji as Docs + UX WG lead

### PR DESCRIPTION
**Pull request by snneji**
# Changes

Update Docs + UX working group lead

### CODE REVIEW

**Review by evankanderson — COMMENTED** [review]
/lgtm
/approve

**Comment by julz** [comment]
@evankanderson I think you need to appease the cla bot too

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -87,6 +92,30 @@ aliases:
   - spencerdillard
   - thisisnotapril
   - vaikas
+  knative-milestone-maintainers:
```
Comment (vaikas) [inline OWNERS_ALIASES:95]: I don't think we want this, and I'm curious why does it keep coming back? :)

File: knative-sandbox-OWNERS_ALIASES [hunk 2]
```diff
@@ -157,6 +175,28 @@ aliases:
   - rhuss
   - thisisnotapril
   - vaikas
+  knative-milestone-maintainers:
```
Comment (vaikas) [inline knative-sandbox-OWNERS_ALIASES:178]: And this.

**Review by vaikas — COMMENTED** [review]
Can you remove knative-milestone-maintainers please?

File: OWNERS_ALIASES (hunk 1 above)
Comment (benmoss) [reply OWNERS_ALIASES:95]: People can't run codegen, I fixed up this PR and submitted #845

**Comment by benmoss** [comment]
/close
#845 fixed this up

**Pull request closed without merging**